File: 8_curso_poo_uber/python/uberVan.py
<<<<<<< HEAD
from car import Car

class uberVan(Car):
    typeCarAccepted = []
    seatsMaterial = []

    def __init__(self, license, driver, typeCarAccepted, seatsMaterial):
        super().__init__(license, driver)
        self.typeCarAccepted = typeCarAccepted
=======
from car import Car

class uberVan(Car):
    typeCarAccepted = []
    seatsMaterial = []

    def __init__(self, license, driver, typeCarAccepted, seatsMaterial):
        super().__init__(license, driver)
        self.typeCarAccepted = typeCarAccepted
>>>>>>> d1bff8aa9e084cc099db80a3377014aa5d90fe12
        self.seatsMaterial = seatsMaterial

File: 8_curso_poo_uber/java/UberVan.java
<<<<<<< HEAD
import java.util.ArrayList;
import java.util.Map;

class UberVan  extends Car{
    Map<String, Map<String, Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;

    public UberVan(String license, Account driver,Map<String, 
    Map<String, Integer>> typeCarAccepted,
    ArrayList<String> seatsMaterial){

        super(license, driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;

    }
    @Override
    public void setPassenger(Integer passenger) {
        if(passenger == 6){
            this.passenger = passenger;
        }else{
            System.out.println("Necesitas asignar 4 pasajeros");
        }
    }
    @Override
    void printDataCar() {
        super.printDataCar();
    }
=======
import java.util.ArrayList;
import java.util.Map;

class UberVan  extends Car{
    Map<String, Map<String, Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;

    public UberVan(String license, Account driver,Map<String, 
    Map<String, Integer>> typeCarAccepted,
    ArrayList<String> seatsMaterial){

        super(license, driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;

    }
    @Override
    public void setPassenger(Integer passenger) {
        if(passenger == 6){
            this.passenger = passenger;
        }else{
            System.out.println("Necesitas asignar 4 pasajeros");
        }
    }
    @Override
    void printDataCar() {
        super.printDataCar();
    }
>>>>>>> d1bff8aa9e084cc099db80a3377014aa5d90fe12
}